client/src/components/_common/InfiniteScrollTable/InfiniteScrollTable.scss

```diff
@@ -6,7 +6,6 @@
   /* FAQ: Table is wider than container when row content is wider than table */
   /* CAVEAT: All tables' column widths must be % values whose sum is 100% */
   /* SEE: https://stackoverflow.com/a/6601257 */
-  table-layout: fixed;
```

Comment:
__This Change Causes a Problem__
_I apologize for not being clear when I described what I knew about the problem with Site Search._

This style `table-layout: fixed;` is still necessary. Please, do not delete it. Comments above the style describe it. A layout bug returns if this style is deleted.

Compare Dashboard on a narrow screen:
- [before deletion](https://user-images.githubusercontent.com/62723358/120047965-8c72aa00-bfdb-11eb-8e19-359881d0c495.png) (__zero__ horizontal table scrollbars)
- [after deletion](https://user-images.githubusercontent.com/62723358/120047984-94324e80-bfdb-11eb-9529-92ddd4ef9514.png) (__two__ horizontal table scrollbars)

Reply:
Thanks for clarifying this. I did not notice the horizontal scroll bars until you pointed them out. Option **C** seems more purposeful. I'll revert this change and create a modifier class to be used where we need the fixed table layout.

Reply:
Ah, typo. Maybe I should not review PRs late on Friday.

> Solution C would [not] be required of you in this PR; just brainstorming for the future.